src/CommunityToolkit/Xamarin.CommunityToolkit/Views/StateLayout/StateLayoutController.shared.cs

```diff
@@ -11,7 +11,7 @@ public class StateLayoutController
 	{
 		readonly WeakReference<Layout<View>> layoutWeakReference;
 		bool layoutIsGrid;
-		LayoutState? previousState;
+		LayoutState? previousState = LayoutState.None;
```

Comment:
Should it be nullable at all than?

Reply:
because it is assigned here https://github.com/xamarin/XamarinCommunityToolkit/blob/e9d6e2440641f805cab4556eb87ffdf30389cff5/src/CommunityToolkit/Xamarin.CommunityToolkit/Views/StateLayout/StateLayoutController.shared.cs#L71

I believe if `newValue` is not null in the following code than yes:
https://github.com/xamarin/XamarinCommunityToolkit/blob/e9d6e2440641f805cab4556eb87ffdf30389cff5/src/CommunityToolkit/Xamarin.CommunityToolkit/Views/StateLayout/StateLayout.shared.cs#L74

https://github.com/xamarin/XamarinCommunityToolkit/blob/e9d6e2440641f805cab4556eb87ffdf30389cff5/src/CommunityToolkit/Xamarin.CommunityToolkit/Views/StateLayout/StateLayout.shared.cs#L93

Reply:
`state` shouldn't be `null` according to it's type (it doesn't have `?`). So `previousState` also shouldn't be.
(`newValue` also shouldn't be null according to the type)